sources/packages/web/src/views/institution/LocationStudents.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <v-container>
+    <h2>{{ locationName }}</h2>
+    <v-sheet elevation="1" class="mx-auto">
+      <v-container>
+        <v-row>
+          <v-col cols="8">
+            <h2 class="color-blue">Students</h2>
+          </v-col>
+        </v-row>
+      </v-container>
+    </v-sheet>
+  </v-container>
+</template>
+
+<script lang="ts">
+export default {
+  props: {
+    locationId: {
+      type: Number,
+      required: true,
+    },
+    locationName: {
+      type: String,
+      required: true,
+    },
+    setup(props: any) {
+      const locationName = props.locationName;
```

Comment:
props can directly be accessed in vue, since locationName in line 28 is not returned in setup, It doesn't have any effect.
i think `setup`  is not required here

Reply:
done